app/lib/forget_member.rb

```diff
@@ -0,0 +1,56 @@
+
+class ForgetMember
+  attr_reader :member
+
+  def self.forget(member)
+    new(member).forget
+  end
+
+  def initialize(member)
+    @member = member
+  end
+
+  def forget
+    anonymise_actions
```

Comment:
Perhaps wrap in TX, currently this could leave you with a semi-anonymised member if something breaks halfway through.